### PR TITLE
H3 aggregation

### PR DESCRIPTION
**Pull request by alainbodiguel**
Adding H3 to collection params before implementing aggregations. Please evaluate if I did not forget something.

### CODE REVIEW

File: arlas-core/src/main/java/io/arlas/server/core/model/CollectionReferenceParameters.java [hunk 1]
```diff
@@ -47,6 +47,9 @@
     @JsonProperty(value = "centroid_path", required = true)
     public String centroidPath;
 
+    @JsonProperty(value = "h3_path", required = true)
```
Comment (sylvaingaudan) [inline arlas-core/src/main/java/io/arlas/server/core/model/CollectionReferenceParameters.java:50]: Having h3 is optional for a collection so the field should also be optional isn't it?

File: arlas-tests/src/test/resources/results/_describe_base_result.json [hunk 2]
```diff
@@ -91,6 +91,93 @@
...
+              "indexed": true
+            },
+            "15": {
+              "type": "KEYWORD",
+              "taggable": false,
+              "indexed": true
+            }
+          },
+          "taggable": false,
+          "indexed": true
```
Comment (sylvaingaudan) [inline arlas-tests/src/test/resources/results/_describe_base_result.json:180]: Is this correct to have H3 indexed and taggable? isn't it an intermediate json node?

Comment (alainbodiguel) [reply arlas-tests/src/test/resources/results/_describe_base_result.json:180]: I tried this mapping:
```json
        "h3": {
          "enabled": false,
          "properties": {
            "0": {
              "type": "keyword",
              "index": true
            },
```
expecting
```
...h3={indexed=false, taggable=false, type=OBJECT, properties={11={indexed=true, taggable=false, type=KEYWORD}...
```
as a result but got 
```
...h3={indexed=false, taggable=false, type=OBJECT, properties={11={indexed=false, taggable=false, type=KEYWORD}...
```

so unless there is another way to skip the parent node, I fear it is not possible.

File: arlas-core/src/main/java/io/arlas/server/core/model/CollectionReferenceParameters.java (hunk 1 above)
Comment (alainbodiguel) [reply arlas-core/src/main/java/io/arlas/server/core/model/CollectionReferenceParameters.java:50]: ok, done.

**Review by mbarbet — APPROVED** [review]

**Review by MohamedHamouGisaia — APPROVED** [review]

**Pull request merged**